config/routes.rb

```diff
@@ -41,19 +41,22 @@
   get '/:run/compare/:comparison_run', to: 'runs#compare',  as: :compare
   get '/:run/download/:timer',         to: 'runs#download', as: :download
 
-  get '/u/:user', to: redirect('/users/%{user}') # deprecated; use GET /users/:user
-
   get '/auth/patreon',          to: 'patreon_users#out'
   get '/auth/patreon/callback', to: 'patreon_users#in'
   get '/auth/patreon/unlink',   to: 'patreon_users#unlink'
 
+  get '/auth/google/callback', to: 'google_users#in'
+  get '/auth/google/unlink',   to: 'google_users#unlink'
+
+  get '/auth/twitch/callback', to: 'twitch_users#in'
+  get '/auth/twitch/unlink',   to: 'twitch_users#unlink'
+
   get '/auth/failure', to: 'sessions#failure'
-  get '/auth/:provider/callback', to: 'sessions#create'
 
-  resources :sessions, only: [:destroy]
```

Comment:
This removes the named path `session_path` which causes the `application.slim` to 500 on line 87.  Adding back `as: :session` seems to fix it